core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1031,20 +1030,46 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def addGoogleProjectLabel(googleProject: GoogleProjectId, key: String, value: String): Future[Unit] = {
```

Comment:
Could we refactor this to take a `Map[String, String]` of labels so that we can add several at once and only make 1 call to Google?